README.md

```diff
@@ -55,8 +55,12 @@ The quantiles of `values` concatenated with `other_values` are still accurate to
 
 ## Development
 
-To work on ddsketch a Python interpreter must be installed. It is recommended
-to create and use a virtualenv for the tooling:
+To work on ddsketch a Python interpreter must be installed. It is recommended to use the provided development
+container (requires [docker](https://www.docker.com/)) which includes all the required Python interpreters.
+
+    docker-compose run dev
```

Comment:
we won't have `riot` available right?

Reply:
Correct, the next step below this is to install riot

Reply:
👍🏻 wasn't sure if we wanted to call that out here or rely on the next steps, seems fine as-is to me